rootfs-builder/rootfs.sh

```diff
@@ -16,6 +16,7 @@ AGENT_BIN=${AGENT_BIN:-kata-agent}
 AGENT_INIT=${AGENT_INIT:-no}
 KERNEL_MODULES_DIR=${KERNEL_MODULES_DIR:-""}
 OSBUILDER_VERSION="unknown"
+export GOPATH=${GOPATH:-${PWD}/go}
```

Comment:
I wonder if this should be `$HOME/go` if `$GOPATH` isn't set? Is `$HOME` guaranteed to be set in a package build environment though I wonder?

Reply:
@jodh-intel I think it could work, also in the case of @lsm5 specfile the gopath is set, I added this because failed as part of my local test, I can move to another PR if needed.

Reply:
I think the fix is fine on its own commit. Any thoughts though @marcov?